### ISSUE TITLE
MDN developer.mozilla.org recipe is disabled for now

### ISSUE THREAD

**Issue by benoit74** [openzim/zim-requests#1044]
Due to upstream bug https://github.com/openzim/warc2zim/issues/293, I disabled MDN recipe at https://farm.openzim.org/recipes/developer.mozilla.org_en until the bug is fixed.

**Comment by benoit74** [comment]
Yes, but still not released ... waiting for that. Plus there is another issue: https://github.com/openzim/warc2zim/issues/327, but I have a hack to workaround this issue, I will probably create a ZIM with the hack from my own machine. 

Thank you for your interest!

**Comment by maevii** [comment]
Hi, just wondering what the status of this is? Both this and the upstream issues haven't seen any activity in around around a year.

FWIW I think the issue with results is minor enough that I wouldn't mind that not working for the time being, at least enabling the recipe to run periodically would be nice.
